docs/GETTING_STARTED_ADD_PAT_OWNER_CLI.md

```diff
@@ -0,0 +1,268 @@
+
+```bash
+"variableGroups": [
+    { "id": 0 },
+    { "id": 0 }
+],
+```
+
+Save the file. Use the az devops invoke command to update the pipeline build definition with the linked variable groups.
+
+```bash
+az devops invoke --http-method PUT --area build --resource definitions --route-parameters project="$TEMPLATE_DEVOPS_PROJECT_NAME" definitionId=$PIPELINE_ID --in-file builddef.json
+```
+
+### 5. Keep the Cobalt Templates relevant to your enterprise patterns and run the pipeline
+
+The goal of this step is to remove infrastructure as code Cobalt Templates that owners of the project have no interest in deploying.
+
+#### a. Clone newly created Azure Devops Repo from your organization
+
+Clone the project into a local repo directory
+
+```bash
+git clone <insert-git-repo-url>
+```
+
+#### b. Review devops pipeline YAML
+
+Remove jobName configurations not relevant to your enterprise patterns. If new to Cobalt, we recommend keeping the path to the az_hello_world template as a starter template. Below is an example of a jobName that you may want to remove by simple deleting the section from the file.
+
+```yaml
+configurationMatrix:
+- jobName: az_service_single_region
+terraformTemplatePath: 'infra/templates/az-service-single-region'
+terraformWorkspacePrefix: 'sr'
+environmentsToTeardownAfterRelease:
+- 'devint'
+```
+
+#### c. Review Cobalt Templates
+
+Manually delete template directories not needed for your enterprise. These will most likely reflect the same templates removed from the devops pipeline YAML. (Do not delete 'backend-state-setup' template! We also recommended keeping the 'az-hello-world' Cobalt Template as a starter template.)
+
+![image](https://user-images.githubusercontent.com/10041279/64913136-1d1e2f00-d700-11e9-95cd-9e95c257bcbd.png)
+
+> NOTE: The CI/CD pipeline needs to detect a code change to run the template-specific build and release jobs (in their respective stages). To force the template build and release to run, you may add a `FORCE_RUN` environment variable with a value of `true` to your *Devint Environment Variables* variable group. You may also add a comment or extra line to a TF or Go file within the template in order for the pipeline script to detect a change without adding any additional override flags
+
+#### b. Commit, Run and verify
+
+```bash
+git commit -m "Removed unrelated templates." && git push
+az pipelines run --name "$COBALT_PIPELINE_NAME"
+```
+
+> NOTE: Integration tests running in the release stage of the pipeline may have resource group level naming conflicts if other tests of the same templates are also running or have been persisted in the Azure portal.
```

Comment:
Is there a recommended way to resolve this? If someone does run into the naming conflicts? Maybe we can add that one liner of a recommended approach as well?

Reply:
I'm not sure this warning applies any longer. I believe that the integration tests use a different Terraform workspace, which means that they should have unique randomized slugs. @nmiodice am I thinking of this correctly?

Reply:
The randomized slugs don't thread through to the terraform workspace state file. Doesn't that mean that this resource group level naming conflict is still a risk at least from within the same pipeline? I believe this risk won't go away unless there's a new state file for every iso deployment. @nmiodice

Reply:
Per our discussion, comment was outdated and removed from these docs.

However, closely related issue is now referenced in #255 "Document Known Limitations". (i.e. Document failures to expect when state is locked as a result of running a pipeline with multiple custom linux hosted build agents)